cedar-policy-core/src/transitive_closure/err.rs

```diff
@@ -23,16 +23,16 @@ use thiserror::Error;
 /// identifier for graph nodes and the type returned by `get_key` on the
 /// `TCNode` trait.
 #[derive(Debug, Error)]
-pub enum Err<K: Debug + Display> {
+pub enum TcError<K: Debug + Display> {
     /// Error raised when `TCComputation::EnforceAlreadyComputed` finds that the
     /// TC was in fact not already computed
     #[error("expected all transitive edges to exist, but they don't. {child} is a child of {parent} is a child of {grandparent}, but {grandparent} is not marked as an ancestor of {child}")]
-    TCEnforcementError {
+    MissingTcEdge {
         /// Child entity at fault
         child: K,
         /// Parent entity at fault
         parent: K,
-        /// Grandparent entity at fault. This is a parent of `parent` but not an
+        /// Grandparent entity at fault. This is a parent of [`parent`] but not an
         /// ancestor of `child`.
```

Comment:
if we're making this change for `parent`, we should be consistent and also do it for `child` later in the sentence

Reply:
Done.